src/accelerate/logging.py

```diff
@@ -58,6 +59,8 @@ def get_logger(name: str):
     Args:
         name (`str`):
             The name for the logger, such as `__file__`
+        log_level (`str`, `optional`, defaults to `None`):
```

Comment:
```suggestion
        log_level (`str`, *optional*):
```
Make sure to re-read the doc-writing guide ;-)